src/config.js

```diff
@@ -8,10 +8,20 @@ const development = {
    multicore: false
  },
  prisma: {
-     host:'localhost',
+     host:"localhost",
      debug: true
+ },
+ gm: {
+  // eslint-disable-next-line camelcase
+  image_width:300,
+  // eslint-disable-next-line camelcase
+  image_height:300,
+  // eslint-disable-next-line camelcase
+  image_format:"jpg"
+ },
+ imageserver:{
+   url:"http://localhost:8007/backend.php"
```

Comment:
In a situation where docker-compose is used the containers that are launched together can reference each other by name not do not need to reference local host.  For example this could be changed to http://image/backend.php

Reply:
changed and pushed